code/datums/uplink_item.dm

```diff
@@ -577,11 +577,33 @@ var/list/uplink_items = list()
 
 /datum/uplink_item/ammo/pistol
 	name = "Magazine - 10mm"
-	desc = "An additional 8-round 10mm magazine for use in the syndicate pistol. These subsonic rounds are dirt cheap but are half as effective as .357 rounds."
+	desc = "An additional 8-round 10mm magazine for use in the syndicate pistol, loaded with standard rounds that are weak but inexpensive"
 	reference = "10MM"
 	item = /obj/item/ammo_box/magazine/m10mm
 	cost = 1
 
+/datum/uplink_item/ammo/pistol
+	name = "Magazine - 10mm Armour Piercing"
+	desc = "An additional 8-round 10mm magazine for use in the syndicate pistol, loaded with rounds that are less effective at injuring the target but penetrate protective gear."
```

Comment:
"loaded with rounds that are less effective at injuring the target but penetrate protective gear."

Less effective at injuring the target? You mean injuring unarmored targets! There's a difference as it depends on the situation. I think that should be cleared up! I know it can be read the 'right' way, but it's still somewhat ambiguous.